src/channeldef.ts

```diff
@@ -104,7 +114,7 @@ export function isConditionalSelection<T>(c: Conditional<T>): c is ConditionalSe
   return c['selection'];
 }
 
-export interface ConditionValueDefMixins<V extends Value = Value> {
+export interface ConditionValueDefMixins<V extends ValueOrGradient = ValueOrGradient> {
```

Comment:
Same here

> I'm wondering if the default should be = ValueOrGradient or = Value? I'm a bit worried that this might lead to mistakes when we add more channels. That said, if we have to override the default in too many internal codes, I'm fine with this as the default.